regression/ebmc/random-traces/initial_state1.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+initial_state1.v
+--random-trace --trace-steps 1 --numbered-trace
```

Comment:
I'd suggest to either explicitly add `--random-seed 0` as command line option or add a comment at the bottom that `0` is the default random seed. Else it is not that obvious that random traces will always produce those "random" initial states.